packages/desktop-client/src/components/accounts/TransactionList.js

```diff
@@ -160,6 +161,14 @@ export default function TransactionList({
     return newTransaction;
   }, []);
 
+  let onManagePayees = useCallback(
+    id => {
+      debugger;
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏need to remove this
```suggestion
```